index.mjs

```diff
@@ -9,17 +13,17 @@ const repoUrl = process.argv[2];
 const daysAmount = process.argv[3];
 
 // Third argument passed via CLI is the name of the branch to consider
-const branchName = process.argv[4] || 'main';
+const branchName = process.argv[4] || "main";
 
-const tmpDir = path.join(import.meta.url.replace('file://', '').replace('index.mjs', ''), 'tmp');
-const resultsDir = path.join(import.meta.url.replace('file://', '').replace('index.mjs', ''), 'results');
+const tmpDir = os.tmpdir() + "tmp-spyone";
+
+const resultsDir = os.tmpdir() + "/results";
 
 if (!repoUrl || !daysAmount) {
-  console.error('Usage: node index.mjs <repo-url> <commit-count>');
+  console.error("Usage: node index.mjs <repo-url> <commit-count>");
```

Comment:
Shouldn't it be "Usage: npx @jointly/spyone <repo-url> <days-to-cover> <branch-name>"?
It was wrong even in the prev version :P

Reply:
Yess 😆